stripe/src/main/java/com/stripe/android/view/AddPaymentMethodActivity.kt

```diff
@@ -75,11 +79,21 @@ open class AddPaymentMethodActivity : StripeActivity() {
 
     private fun configureView(args: AddPaymentMethodActivityStarter.Args) {
         viewStub.layoutResource = R.layout.add_payment_method_layout
-        val contentRoot = viewStub.inflate() as ViewGroup
+        val scrollView = viewStub.inflate() as ViewGroup
+        val contentRoot = scrollView.getChildAt(0) as ViewGroup
 
         addPaymentMethodView = createPaymentMethodView(args)
         contentRoot.addView(addPaymentMethodView)
 
+        if (args.addPaymentMethodFooter > 0) {
+            val footerView = layoutInflater.inflate(args.addPaymentMethodFooter, contentRoot, false)
+            if (footerView is TextView) {
+                LinkifyCompat.addLinks(footerView, Linkify.ALL)
```

Comment:
I wonder if we can drop this and let the user specify it via XML? I'm thinking of a scenario where the user doesn't want something linkified

Reply:
In testing various scenarios, the XML autoLink didnt work for some of them.  This combo seems to work for all the ones I could think of.

For example where autoLink fails `<a href="https://stripe.com">Stripe</a>` but it would work if you had the URL in the string.

To get around us auto linking, if they wrap their footer text view in a view group like frame layout, it would apply this since the root child isn't a textview

Reply:
thanks for clarifying!